README.rst

```diff
@@ -92,3 +92,24 @@ __ https://docs.python.org/3/library/functools.html#functools.partial
 
         def handle(self, *args, **options):
             ...
+
+``django_rich.test.RichRunner``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A subclass of Django's |DiscoverRunner|__ with colourized outputs and `nice traceback rendering <https://rich.readthedocs.io/en/stable/traceback.html>`__.
+
+.. image:: img/RichRunner.png
+
+.. |DiscoverRunner| replace:: ``DiscoverRunner``
+__ https://docs.djangoproject.com/en/stable/topics/testing/advanced/#defining-a-test-runner
+
+To use the runner, point your add a |TEST_RUNNER|__ setting to it:
+
+.. |TEST_RUNNER| replace:: ``TEST_RUNNER``
+__ https://docs.djangoproject.com/en/stable/ref/settings/#std:setting-TEST_RUNNER
+
+.. code-block:: python
+
+    TEST_RUNNER = "django_rich.test.RichRunner"
+
+You can also use it for your custom subclasses.
```

Comment:
I'd like to cover more of the benefits here, like revealing of locals, how may terminals let you click (or ctrl/command-click) on file:lineno lines to open them that file at that line, syntax highlighting